src/fabric/ScreenNativeComponent.js

```diff
@@ -19,6 +19,13 @@ type ScreenDismissedEvent = $ReadOnly<{|
   dismissCount: Int32,
 |}>;
 
+type GestureResponseDistanceType = $ReadOnly<{|
+  start: Float,
```

Comment:
Will it work if you do not apply e.g. `start` ?

Reply:
As I could not find a way to make codegen generate default value for nested structure I [added setting of default value to JS code](https://github.com/software-mansion/react-native-screens/pull/1442/commits/c0413bddadc1023022d3d390373321632ad3539d) (it is so for both architectures now) but it has still to be checked if this does not ruin any JS code.